what4-transition-system/what4-transition-system.cabal

```diff
@@ -17,7 +17,6 @@ common dependencies
     , bytestring
     , containers           ^>=0.6
     , io-streams
-    , language-sally       ^>=0.3
```

Comment:
Make sure to also remove the references to `language-sally` in the following places:

* https://github.com/GaloisInc/what4/blob/ea717ac94a186b5ee18f138b71d8b4b4b2f00955/cabal.project#L17
* https://github.com/GaloisInc/what4/blob/ea717ac94a186b5ee18f138b71d8b4b4b2f00955/.gitmodules#L10-L12

There's also a test case in `what4-transitions-system` that imports Sally-specific modules:

* https://github.com/GaloisInc/what4/blob/ea717ac94a186b5ee18f138b71d8b4b4b2f00955/what4-transition-system/test/Main.hs#L25
* https://github.com/GaloisInc/what4/blob/ea717ac94a186b5ee18f138b71d8b4b4b2f00955/what4-transition-system/test/Main.hs#L36

I doubt that this still compiles after these changes, so it would be worthwhile to migrate the parts of that test suite that depend on Sally over to `language-sally`.

Reply:
Good catch!